utils/kernel-refresh.sh

```diff
@@ -1,26 +1,32 @@
 #!/bin/sh
 
 KDIR=$1
-KTAG=$2
-PDIR=$3
-
+OLD_VER=$2
+KTAG=$3
+PDIR=$4
+DDIR=$5
+NEW_VER=$6
 usage()
 {
     cat <<EOF
-usage: $0 <kernel-dir> [<kernel-tag>] [<patch-dir>]
+usage: $0 <kernel-dir> <old-version> [<kernel-tag>] [<patch-dir>] [defconfig-dir] [new-version]
 
 Synchronize patches directory with changes from a GIT versioned kernel
 tree.
 
   <kernel-dir>  Path to kernel tree.
-
+  <old-version> Version to be replaced by new kernel version.
   <kernel-tag>  Base tag from which to generate patches. Default: "v"
                 followed by the value of BR2_LINUX_KERNEL_VERSION in
                 the current configuration.
 
   <patch-dir>   Path to kernel patches directory. Default: the value of
                 BR2_EXTERNAL_INFIX_PATH in the current configuration,
-                followed by "/patches/linux".
+                followed by "/patches/linux/<kernel-tag>".
+  <defconfig-dir> Path to defconfig. Default the value of BR2_EXTERNAL_INFIX_PATH
+  		  in the current configuration, followed by "/configs".
+  <new-version> New version of kernel. Default: kernel-tag with removed v
```

Comment:
When do we expect this to be different from `<kernel-tag>`?

The original use-case was:
1. Update the version in your config
2. Run `kernel-refresh.sh`

Its perfectly fine if we want to change this (since you now update all defconfigs, which is great!), but it seems like `<kernel-tag>` and `<new-version>` are intimately linked. Can we not infer one from the other?